k4EDM4hep2LcioConv/src/k4EDM4hep2LcioConv.cpp

```diff
@@ -147,4 +147,14 @@ std::unique_ptr<lcio::LCEventImpl> convertEvent(const podio::Frame& edmEvent, co
   return lcioEvent;
 }
 
+std::optional<double> getRadiusOfStateAtFirstHit(const edm4hep::Track& track, bool use3D) {
```

Comment:
The `use3D` parameter as it is will never be used other than in the test below? I guess it doesn't hurt but for someone reading the code it could look like you can have the two ways when the converter defaults to one and there isn't a way of setting the other.